EIPS/eip-1820.md

```diff
@@ -921,7 +921,7 @@ Copyright and related rights waived via [CC0].
 [ERC165 Cache]: #erc165-cache
 [Nick's article]: https://medium.com/@weka/how-to-send-ether-to-11-440-people-187e332566b7
 [0xjac/ERC1820]: https://github.com/0xjac/ERC1820
-[CC0]: https://creativecommons.org/publicdomain/zero/1.0/
+[CC0]: ../CC0
```

Comment:
We should move this up into the Copyright section.  We don't allow bibliographies like this anymore, but this EIP was grandfathered in.  Since we are adjusting all licenses, we should adjust this one to match at least that section.